src/main/java/frc/robot/subsystems/intake/LocateCoral.java

```diff
@@ -19,7 +19,7 @@ public class LocateCoral extends Command {
 
   private Supplier<CoralEnum> state;
   private IntakeSubsystem intakeSubsystem;
-  private BooleanSupplier elevatorNotL1;
+  public BooleanSupplier elevatorNotL1;
```

Comment:
We are going to repurpose this one to override the uninterruptible Command, so leave it as private and mark as a TODO